README.md

```diff
@@ -25,29 +33,42 @@ docker run -p 27017:27017 -v $PWD/data/mongodb:/data/db -d mongo:3.6.6-jessie --
 Import into mongodb, e.g. for Vienna dataset:
 
 ```
-npm run import configWien/config-import-schedule.json
+npm run import configWien/config-Wien.json
 ```
 
 ### Extract GeoJSON files from data in mongodb
 
-* Extract GeoJSON files of all stops: `npm run getStops PATH_TO_CONFIG_FILE`
-* Extract GeoJSON files of schedules: `npm run getSchedule PATH_TO_CONFIG_FILE`
-
-Sample config files are located in the `Netherlands/` and `Wien/` directories.
+The config files here are an extension of the config file for the data import in mongodb (See configNetherlands/config-Netherlands.json for an example)
+* ```node node node_modules/gtfs-to-geojson/bin/gtfs-to-geojson.js --configPath PATH\_TO\_CONFIG\_FILE  --skipImport```
```

Comment:
I think you don't need to (aka shouldn't) escape underscores here and on the lines below